src/test/java/org/opentripplanner/routing/algorithm/filterchain/groupids/GroupByAllSameStationsTest.java

```diff
@@ -75,7 +76,7 @@ public void testMatchingItineraries() {
 
 
     Station createStation(String name) {
-        return new Station(new FeedScopedId(FEED_ID, name), name, new WgsCoordinate(0, 0), null, null, null, null, null);
+        return new Station(new FeedScopedId(FEED_ID, name), new NonLocalizedString(name), new WgsCoordinate(0, 0), null, null, null, null, null);
```

Comment:
Should we extract this functionality to the `Station`, `Stop` etc constructors, so that the `NonLocalizedString` gets instantiated there, if we supply a `String` instead of `I18NString` to it?

Reply:
Done.